docker/release.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+error() {
+    echo $@ >&2
+    return 1
+}
+
+if [[ $# -ne 1 ]]; then
+    error $0 [hash\|tag\|branch]
+fi
+
+if [[ -d tikv ]]; then
+    cd tikv
+    if [[ ! -d .git ]]; then
+        git init .
+        git remote add origin https://github.com/pingcap/tikv.git
+    fi
+    git fetch origin
+else
+    git clone https://github.com/pingcap/tikv.git
+    cd tikv
+fi
+git checkout $1
```

Comment:
I think we don't need provide the git branch or hash by args.  However, just checkout to the specified branch, and run this release-docker to compile tikv-server, remember that docker run with --volume ${PWD}:/tikv